pkg/worker/redis.go

```diff
@@ -96,7 +83,11 @@ func (w *RedisWorker) Enqueue(ctx context.Context, job *Job) error {
 		}
 	}
 
-	logger := loggerWithJob(ctx, job)
```

Comment:
Why did you removed the function? :thinking:

Reply:
Yeah there is replacement now, only for two cases there is not yet enough information so I replaced it with plain code.